### PR TITLE
build(docker): refactor ingestion docker build script

### PR DESCRIPTION
**Pull request by mars-lan**
- add "build" option to docker-compose file to simplify rebuilding of images
- move command from docker-compose.yml to Dockerfile
- add ingestion.sh script to simplify quickstart instruction and to reduce confusion

Fix https://github.com/linkedin/datahub/issues/1689

## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: docker/ingestion/README.md [hunk 1]
```diff
@@ -3,18 +3,13 @@
...
-This command will build and deploy the image in your local store.
+This command will rebuild the docker image and start a container based on the image.
 
-## Run container
-```
- cd docker/ingestion && docker-compose up
-```
-This command will start the container. If you have the image available in your local store, this image will be used
-for the container otherwise it will build the image from local repository and then start that.
+To start a container using an existing image, run the same command without the `--build` flag.
```
Comment (keremsahin1) [inline docker/ingestion/README.md:12]: Same comment as in the other PR: https://github.com/linkedin/datahub/pull/1687

Comment (mars-lan) [reply docker/ingestion/README.md:12]: We never publish this image so will need to always build it

Comment (keremsahin1) [reply docker/ingestion/README.md:12]: Oh haha correct :)

**Comment by mars-lan** [comment]
> We can get rid of the command in docker-compose.yml now that we have it in Dockerfile?
> (Interestingly Github didn't allow me to add this comment on that specific line, so adding it as a general comment)

Done

**Pull request merged**